db/migrate/20160719170253_add_biography_to_profile.rb

```diff
@@ -0,0 +1,5 @@
+class AddBiographyToProfile < ActiveRecord::Migration
+  def change
+    add_column :profiles, :biography, :text, default: ""
```

Comment:
I think it's better to leave the default as NULL, so that you can distinguish between an empty profile and one that wasn't filled in yet. This would defeat validations for presence for example.